functions/supplier/http-supplier/pom.xml

```diff
@@ -28,7 +28,6 @@
         <dependency>
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-configuration-processor</artifactId>
-            <scope>provided</scope>
```

Comment:
But it is indeed `provided` in other places...
What made you to remove it over here?
Thanks

Reply:
looks like a mistake